src/psyclone/dynamo0p3.py

```diff
@@ -8058,11 +8007,14 @@ def stencil_unknown_extent(self, arg):
         '''
         Add stencil information associated with a kernel argument if the
         extent is unknown.
+
         :param arg: the meta-data description of the kernel argument with \
                     which the stencil is associated.
         :type arg: :py:class:`psyclone.dynamo0p3.DynKernelArgument`
         '''
-        name = DynStencils.dofmap_size_name(arg)
+        # When generating stubs we have kernels that are not attached
+        # to an InvokeSchedule, we use a dummy SymbolTable then.
```

Comment:
We should probably use a symbol table for stubgen which would avoid accidental name clashes but I appreciate that is down the line.

Reply:
I made a common self._stub_symtab for all the KernStubArgList, but some other parts of the Stub are still disconnected. I made #719 to reference this issue.

Reply:
Great.